### ISSUE TITLE
sql: TestExecBuild/local-opt/show_trace failed

### ISSUE THREAD

**Issue by tbg** [cockroachdb/cockroach#35093]
```
TestExecBuild/local-opt/show_trace

=== PAUSE TestExecBuild/local-opt/show_trace
            logic.go:2313: 
                 
                testdata/show_trace:197: SELECT operation, regexp_replace(message, '(\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.)?\d\d\d\d\d+', '...PK...') as message
                  FROM [SHOW KV TRACE FOR SESSION]
                WHERE message NOT LIKE '%Z/%'
                  AND tag NOT LIKE '%intExec=%'
                  AND tag NOT LIKE '%scExec%'
                  AND tag NOT LIKE '%IndexBackfiller%'
                expected:
                    dist sender send  querying next range at /Table/2/1/53/"kv2"/3/1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  querying next range at /Table/3/1/55/2/1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r7: sending batch 1 EndTxn, 1 QueryIntent to (n1,s1):1
                    table reader      Scan /Table/55/{1-2}
                    dist sender send  querying next range at /Table/55/1
                    dist sender send  r20: sending batch 1 Scan to (n1,s1):1
                    table reader      fetched: /kv2/primary/...PK.../k/v -> /1/2
                    flow              Put /Table/55/1/...PK.../0 -> /TUPLE/1:1:Int/1/1:2:Int/4
                    dist sender send  querying next range at /Table/55/1/...PK.../0
                    dist sender send  r20: sending batch 1 Put, 1 EndTxn to (n1,s1):1
                    flow              fast path completed
                    sql txn           rows affected: 1
                    
                but found (query options: "") :
                    dist sender send  querying next range at /Table/2/1/53/"kv2"/3/1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  querying next range at /Table/3/1/55/2/1
                    dist sender send  r6: sending batch 1 Get to (n1,s1):1
                    dist sender send  r7: sending batch 1 EndTxn, 1 QueryIntent to (n1,s1):1
                    table reader      Scan /Table/55/{1-2}
                    dist sender send  querying next range at /Table/55/1
                    dist sender send  r20: sending batch 1 Scan to (n1,s1):1
                    dist sender send  querying next range at /Table/SystemConfigSpan/Start
                    dist sender send  r6: sending batch 1 PushTxn to (n1,s1):1
                    table reader      fetched: /kv2/primary/...PK.../k/v -> /1/2
                    flow              Put /Table/55/1/...PK.../0 -> /TUPLE/1:1:Int/1/1:2:Int/4
                    dist sender send  querying next range at /Table/55/1/...PK.../0
                    dist sender send  r20: sending batch 1 Put, 1 EndTxn to (n1,s1):1
                    flow              fast path completed
                    sql txn           rows affected: 1
                    
                
            logic.go:2345: 
                testdata/show_trace:220: error while processing
            logic.go:2346: testdata/show_trace:220: too many errors encountered, skipping the rest of the input
```

https://teamcity.cockroachdb.com/viewLog.html?buildId=1145560&buildTypeId=Cockroach_UnitTests

**Comment by RaduBerinde** [comment]
The extra lines are:
```
                    dist sender send  querying next range at /Table/SystemConfigSpan/Start
                    dist sender send  r6: sending batch 1 PushTxn to (n1,s1):1
```

This is similar to #34911. The second line would be removed by the fix for that issue. Any ideas about the first? At first glance we should probably ignore that as well?
CC @andy-kimball @nvanbenschoten

**Comment by jordanlewis** [comment]
This is resolved by 92f74b918d6fa63393fe00b33d1272c63b3ebbcf
